Bottom: 47b8185a2289ffd6db2f37f1c75141db7cd43b2c
Top:    7a7687f93805696e882d05ca8927c61c09a1d57c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-14 21:55:33 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
